whois-query/src/main/java/net/ripe/db/whois/query/executor/RpslObjectSearcher.java

```diff
@@ -48,7 +48,8 @@
 class RpslObjectSearcher {
     private static final Logger LOGGER = LoggerFactory.getLogger(RpslObjectSearcher.class);
     private static final Set<AttributeType> INVERSE_ATTRIBUTE_TYPES = EnumSet.noneOf(AttributeType.class);
-    private static final Set<AttributeType> INVERSE_ATTRIBUTE_TYPES_OVERRIDE = EnumSet.of(AttributeType.SPONSORING_ORG);
+
+    private static final Set<AttributeType> INVERSE_ATTRIBUTE_TYPES_OVERRIDE = EnumSet.of(AttributeType.E_MAIL);
```

Comment:
Make sure the other PR doesn't remove the code:
```
if (!(INVERSE_ATTRIBUTE_TYPES.contains(attributeType) || (query.isTrusted() && INVERSE_ATTRIBUTE_TYPES_OVERRIDE.contains(attributeType)))) {
...
```

Reply:
other PR will be merged BEFORE this one
This will contain last changes

Reply:
is there after merging